README.md

```diff
@@ -148,6 +148,10 @@ Runs the following tasks
 
 Generate an SVG sprite and copy into the dist directory
 
+### `copy:js`, `copy:css` & `copy:img`
```

Comment:
OCD alert 🚨 — can you move this section below service workers so it matches the position in the other sections?